tests/features/publication.feature

```diff
@@ -4,29 +4,95 @@ Feature: Publication content creation
   As an editor
   I need to be able to create and see publication items
 
+  @javascript
   Scenario: Creation of a Publication content through the UI.
-    Given I am logged in as a user with the "create oe_publication content, access content, edit own oe_publication content, view published skos concept entities" permission
-    # Create a "Document".
+    Given I am logged in as a user with the "create oe_publication content, access content, edit own oe_publication content, view published skos concept entities, manage corporate content entities" permission
+
     And the following documents:
     | name          | file       |
     | My Document 1 | sample.pdf |
-    # Create a "Publication" content.
+
+    And the following images:
+    | name         | file           | alt          |
+    | Sample image | example_1.jpeg | example text |
+
+    And the following General Contact entity:
+    | Name               | A general contact                                                                         |
+    | Address            | country_code: BG - locality: Varna - address_line1: General contact 1 - postal_code: 9009 |
+    | Email              | general@example.com                                                                       |
+    | Phone number       | +359525566778                                                                             |
+    | Social media links | uri: http://instagram.com - title: Instagram - link_type: instagram                       |
+
     And I visit "the Publication creation page"
     And I fill in "Page title" with "My Publication item"
     And I fill in "Introduction" with "Summary text"
     And I fill in "Teaser" with "Teaser text"
     And I fill in "Subject" with "financing"
-    And I fill in "Publication date" with the date "2019-02-21"
-    And I fill in "Use existing media" with "My Document 1"
-    And I fill in "Type" with "Acknowledgement receipt"
+    And I set "Publication date" to the date "21-02-2019"
+    And I fill in "oe_documents[0][target_id]" with "My Document 1"
```

Comment:
We need to use field labels. You could try `I fill in the 1st "Use existing media"` and `I fill in 2nd "Use existing media"` to be able to fill in both the files and the thumbnail fields.

Reply:
This is fixed.